src/chart/Treemap.tsx

```diff
@@ -360,8 +360,9 @@ export class Treemap extends PureComponent<Props, State> {
     e.persist();
     const { onMouseEnter, children } = this.props;
     const tooltipItem = findChildByType(children, Tooltip);
+    const tooltipTrigger = tooltipItem.props.trigger;
```

Comment:
Another `findChildByType` 😢 we need to move all of this to context

Reply:
Outside of this PR of course.

Reply:
yep. I'm solving problems from issues in a siloed way.. need to start helping with the broader changes.